src/NzbDrone.Common/Processes/ProcessOutput.cs

```diff
@@ -5,6 +5,7 @@ namespace NzbDrone.Common.Processes
 {
     public class ProcessOutput
     {
+        public int ExitCode { get; set; }
         public List<String> Standard { get; private set; }
```

Comment:
I think we should merge this into a list of `OutputLine` where it has `Level`, `Content` and `Date`, right now there is no way to see all messages in order.